ingress/controllers/nginx/controller.go

```diff
@@ -89,9 +90,11 @@ type loadBalancerController struct {
 	ingController  *framework.Controller
 	endpController *framework.Controller
 	svcController  *framework.Controller
+	mapController  *framework.Controller
 	ingLister      StoreToIngressLister
 	svcLister      cache.StoreToServiceLister
 	endpLister     cache.StoreToEndpointsLister
+	mapLister      StoreToMapLister
```

Comment:
can you rename to storetoconfigmaplister? since this makes it sounds like you're listing to a go map